src/dto/model.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * DeepDetect
+ * Copyright (c) 2021 Jolibrain SASU
+ * Author: Mehdi Abaakouk <mehdi.abaakouk@jolibrain.com>
+ *
+ * This file is part of deepdetect.
+ *
+ * deepdetect is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * deepdetect is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with deepdetect.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef DTO_MODEL
+#define DTO_MODEL
+
+#include "oatpp/core/Types.hpp"
+#include "oatpp/core/macro/codegen.hpp"
+
+namespace dd
+{
+  namespace DTO
+  {
+#include OATPP_CODEGEN_BEGIN(DTO) ///< Begin DTO codegen section
+
+    class Model : public oatpp::DTO
```

Comment:
`DTO_DD_Model` instead, or something else ? I understand the namespace should be enough, but I find it confusing. Maybe we just need a naming convention for the API objects.